source/fundamentals/crud/write-operations/change-a-document.txt

```diff
@@ -0,0 +1,159 @@
+=================
+Change a Document
+=================
+
+.. default-domain:: mongodb
+
+You can change documents in a MongoDB collection using two distinct
+operation types: :ref:`update <update-operation>` and :ref:`replace
+<replace-operation>`. Update operations change specified fields in one
```

Comment:
I think it's potentially confusing to the user to surface links without explaining what they are. It can be helpful to provide that explanation, but it might not fit so well in your current opening sentence. Consider rewording the opening sentence to identify what the user can learn in this guide/page and identifying them as sections. E.g. 
```
In this guide, you can learn how to change documents in a MongoDB collection using write operation types explained in the following sections:
- :ref:`Update <update-operation>`
...
```